docs/README.md

```diff
@@ -1,59 +1,90 @@
-# WooCommerce internal documentation
+# WooCommerce Developer Documentation
 
-This directory contains documentation about implementation details specific parts of the WooCommerce code base. This documentation is intended for developers.
+> ⚠️ **Notice:*- This documentation is currently a **work in progress**. While it's open to the public for transparency and collaboration, please be aware that some sections might be incomplete or subject to change. We appreciate your patience and welcome any contributions!
```

Comment:
```suggestion
> ⚠️ **Notice:** This documentation is currently a **work in progress**. While it's open to the public for transparency and collaboration, please be aware that some sections might be incomplete or subject to change. We appreciate your patience and welcome any contributions!
```

Needed else this will literally render the asterisks and hyphen instead of making **Notice:** bold.